src/ansys/dpf/core/collection.py

```diff
@@ -25,13 +25,17 @@
 
 from __future__ import annotations
 
+from typing import Generic, Type, TypeVar
+
 from ansys.dpf.core import errors, server as server_module
 from ansys.dpf.core.any import Any
-from ansys.dpf.core.collection_base import TYPE, CollectionBase
+from ansys.dpf.core.collection_base import CollectionBase
 from ansys.dpf.core.common import create_dpf_instance
 
+TYPE = TypeVar("TYPE")
```

Comment:
Let's use `T` as per the original [PEP 484]:


```suggestion
T = TypeVar("T")
```

[PEP 484]: https://peps.python.org/pep-0484/#user-defined-generic-types